src/include/catalog/namespace.h

```diff
@@ -95,4 +95,6 @@ extern char *namespace_search_path;
 
 extern List *fetch_search_path(bool includeImplicit);
 
+#define HiveProfileName "Hive"
```

Comment:
why is it defined here? shouldn't it be in a less general location, as it's only used by pxf/hcatalog?